stdlib/shutil.pyi

```diff
@@ -61,7 +61,12 @@ class _ntuple_diskusage(NamedTuple):
     free: int
 
 def disk_usage(path: int | StrOrBytesPath) -> _ntuple_diskusage: ...
-def chown(path: StrPath, user: str | int | None = ..., group: str | int | None = ...) -> None: ...
+@overload
+def chown(path: StrOrBytesPath, user: str | int, group: None = ...) -> None: ...
+@overload
+def chown(path: StrOrBytesPath, user: None = ..., *, group: str | int) -> None: ...
```

Comment:
There are two ways to specify group without specifying the user: group is a keyword argument (e.g. `chown("dir/file", group="audio")`, handled by your overload), group is a positional argument (not handled by your overload, e.g. `chown("dir/file", None, "audio")`).

```suggestion
def chown(path: StrOrBytesPath, user: None = ..., *, group: str | int) -> None: ...
@overload
def chown(path: StrOrBytesPath, user: None, group: str | int) -> None: ...
```

Reply:
ahh great spot there, thanks! 😄